Modules/Filtering/LabelMap/test/itkLabelImageToLabelMapFilterTest.cxx

```diff
@@ -40,20 +40,13 @@ zeroSizeCase()
 int
 itkLabelImageToLabelMapFilterTest(int argc, char * argv[])
```

Comment:
I have a feeling that some compiler will complain about unused variables `argc` and `argv`. The names should be removed, only leaving the types `itkLabelImageToLabelMapFilterTest(int, char **)`

Reply:
:+1: Addressed in 0515fae.